### PR TITLE
Log when the timeline index is empty

### PR DESCRIPTION
**Pull request by Pchelolo**
cc @wikimedia/services

### CODE REVIEW

File: sys/multi_content_bucket.js [hunk 1]
```diff
@@ -378,7 +378,11 @@ class MultiContentBucket {
                         });
                     })
                     .catch({ status: 404 }, () => {
-                        // Ignore the 404 if we don't have the timeline.
+                        // Log the 404 if we don't have the timeline.
+                        hyper.log('error/noindex', {
+                            msg: 'Empty revision timeline',
+                            page_title: rp.key
```
Comment (eevans) [inline sys/multi_content_bucket.js]: Do we need the domain logged as well, or will that be available elsewhere?

File: sys/multi_content_bucket.js [hunk 2]
```diff
@@ -426,7 +430,12 @@ class MultiContentBucket {
                         });
                     })
                     .catch({ status: 404 }, () => {
-                        // Ignore the 404 if we don't have the timeline.
+                        // Log the 404 if we don't have the timeline.
+                        hyper.log('error/noindex', {
+                            msg: 'Empty render timeline',
+                            page_title: rp.key,
+                            page_revision: rev
```
Comment (eevans) [inline sys/multi_content_bucket.js]: Idem.

File: sys/multi_content_bucket.js (hunk 1 above)
Comment (Pchelolo) [reply sys/multi_content_bucket.js]: The domain should be added by default.

**Comment by eevans** [comment]
LGTM :+1: ; Are these test failures new?  They seem wholly unrelated.

**Comment by d00rman** [comment]
Perhaps `warn` would have been a more appropriate logging level given that we've been ignoring these so far and that, supposedly, a 404 here does not signal an actual failure

**Pull request merged**